bfd/src/main/java/gov/cms/ab2d/bfd/client/BFDClient.java

```diff
@@ -11,8 +11,8 @@
     String BFD_CLIENT_ID = "AB2D";
     String BFD_HDR_BULK_JOBID = "BULK-JOBID";
 
-    IBaseBundle requestEOBFromServer(FhirVersion version, String patientID);
-    IBaseBundle requestEOBFromServer(FhirVersion version, String patientID, OffsetDateTime sinceTime);
+    IBaseBundle requestEOBFromServer(FhirVersion version, Long patientID);
```

Comment:
Can patient IDs ever be null?  

Otherwise, make it a primitive and one avoids tons of NPE issues.